### ISSUE TITLE
Feedback on Week 1 (OL5)

### ISSUE THREAD

**Issue by abbycabs** [MozillaFoundation/mentorship-program#47]
Hello all!
How did you find Week 1: Introductions & Project Planning?

* What went well?
* What could be better?
* Is there anything you'd like to see more or less of during the program?

This program is here to help you! If there's anything you'd like to let us know, this is a great place. You're always welcome to send me an email (abby AT mozillafoundation.org)

**Comment by booyaa** [comment]
- What went well?

> The time keeping was pretty good, I did expect a bit of overrun because we're using stuff that not everyone has familiarity with i.e. etherpad, zoom. Really glad that by default we're on mute, I kind find it really distracting when you can hear background noise.
>
> What was really amazing was the break out sessions, the transition from cohort to smaller groups was super smooth and I found immensely useful!

- What could be better?

> I wish the sessions could be longer, yeah I know 1.5 hours is a big chunk of time for everyone involved. I'm surprised how quickly it flew past.

- Is there anything you'd like to see more or less of during the program?

> Pretty happy with how the programme is going so far.

**Comment by hepplerj** [comment]
- **What went well?**

The breakout sessions were really useful (and introduced me to a feature I had no idea existed in Zoom). I think overall the introduction of all of us and the OL program went smoothly. 

- **What could be better?**

Nothing top of mind right now.

- **Is there anything you'd like to see more or less of during the program?**

Things went great for the first meeting!

**Comment by abbycabs** [comment]
Thanks @booyaa & @hepplerj -- I'm glad you both enjoyed the first meeting & the magic of breakout rooms :) I'm sympathetic to the rushed feeling, but it's tough to get people to commit to more than 90min of video calls. I'll look into spreading out content for future rounds.

I'll leave this open till Week 3, please let us know if there's anything we should tweak for future meetings!

**Comment by lognaturel** [comment]
+1 on breakout sessions being really cool!

I love the idea of a way to share ideas in between sessions and seeing everyone's canvases has been super valuable. I'm finding Gitter to be a challenging platform because it doesn't allow for things like threading or reactions. I'd love to put a thumbs up next to canvases I particularly like or be able to see grouped feedback on a specific canvas. An async platform like Discourse would work better for me personally but chat with threading/reactions would also help. Maybe those features are available in Gitter but need to be turned on?

**Comment by abbycabs** [comment]
Thanks @lognaturel!

We've tried a couple chat platforms in the past and it's tough to find the right balance. I never considered Discourse, but there is an instance we could use! I'll look into it for next round.

We also considered Slack for this round, but didn't sort out the logistics in time.

Very helpful, I appreciate your thoughts here

**Comment by denefi** [comment]
- What went well?
The breakoutsession was really inspiring and the ressources that were offered in the breakoutsession but also in the emails and etherpad were really usefull. It feels great to be part of this community and i think it's a great way to improve my groups proyect, but also personally.

- What could be better?
I find it kind of strange, that so much proprietary is used in the program. I had to creat a Google Account to fill the forms and later to access program resources. The same for zoom. My personal excuse to use so many of those proprietary tools was allways that i have no server on my own. But i think that one doesn't work for Mozilla ;)

**Comment by abbycabs** [comment]
@denefi great question! And one that I've struggled with for quite awhile. It's possible to find open source alternatives for everything, but it was hard to get people to use something they weren't used to or was difficult to install. I ended up with the philosophy that I want to meet people where they are (which is often google, github, etc), and focus on teaching open practices and processes (with whatever tools you have). This makes it a bit easier for people to participate, and helps us have a bigger impact.

My hope is that one day, this impact will affect the tools we use day-to-day. And soon, we'll have even better open-source alternatives to the proprietary tools we rely on 😄 

(I will note that it is *possible*, albeit quite difficult to go through the program without a google account. We had one project lead do it in round 3)
